pkg/controller/installation/core_controller.go

```diff
@@ -1365,6 +1366,9 @@ func (r *ReconcileInstallation) Reconcile(ctx context.Context, request reconcile
 	}
 
 	// Write updated status.
+	if statusMTU > math.MaxInt32 || statusMTU < math.MinInt32 {
```

Comment:
Instead of checking greater than MinInt32 we should just check for >= 0 because a negative MTU is invalid.

Reply:
Fixed